website/docs/r/consul_secret_backend.html.md

```diff
@@ -38,7 +38,7 @@ The following arguments are supported:
   The `namespace` is always relative to the provider's configured [namespace](../index.html#namespace).
    *Available only for Vault Enterprise*.
 
-* `token` - (Required) The Consul management token this backend should use to issue new tokens.
+* `token` - (Optional) The Consul management token this backend should use to issue new tokens. If unspecified, Vault will attempt to perform an ACL Bootstrap automatically on the Consul node.
```

Comment:
This is still required with Vault-1.10 and below right?

If you could break on the 100 column mark that would great as well.

Reply:
That is right, I shouldn't have forgotten that.